api-client/src/maps/index.ts

```diff
@@ -11,6 +11,18 @@ export type MapViewViewport = {
   zoom: number
 }
 
+/**
+ * Override existing GeoJsonProperties from types/geojson to be more precise
+ */
+export type GeoJsonPoiProperties = {
```

Comment:
Maybe we could create aliases for `Feature<Point, GeoJsonPoiProperties>` and `FeatureCollection<Point, GeoJsonPoiProperties` as well? We are using them really often and I think simpler types would be nice

Reply:
is that what you have expected?

```js
// aliases for Features and FeatureCollections using custom GeoJsonProperties and Point
export type PoiFeature = Feature<Point, GeoJsonPoiProperties>
export type PoiFeatureCollection = FeatureCollection<Point, GeoJsonPoiProperties>
```

Reply:
Yep, would be nice!